windows/tmpl.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/nanoserver:$WINDOWS_VERSION
```

Comment:
Wouldn't `BUILD-ARGS` be better for that case (and not having `tmpl` stuff).

```
ARG version sac2016
FROM microsoft/nanoserver:${version}

#…
```

Same for `$VERSION` :angel:

Reply:
I'm not sure that build-arg is supported in https://github.com/docker-library/official-images?

Reply:
Yeah, sadly, I think `BUILD-ARGS ` are not supported, neither on official images nor on Docker Hub.

Reply:
:scream: